CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 
 
 * Python 2.7 wheels for Windows are no longer being built (#71, 73).
+* CRC and size validation of uncompressed data is now performed by default, on
+  the first pass through a GZIP file. This is not yet implemented when an
```

Comment:
Just making sure -- would null padding bytes still not cause a problem if an existing index is imported from a file?

Reply:
No, padding bytes are skipped over regardless of whether CRC validation takes place or not. I'm disabling CRC validation for indexes imported from file for the time being, because it would require a change to the index file format, and I didn't design the format, nor think to build in space for future expansion. 

And, now that I think about it, there's no way to validate the CRC unless all of the data is read and decompressed (i.e. so we can calculate our own CRC to validate against), which is what we use a pre-generated index file to avoid having to do! So maybe CRC validation will always be disabled for pre-generated indexes.